CMakeLists.txt

```diff
@@ -7,7 +7,7 @@ set(CMAKE_CXX_FLAGS_DEBUG "-Werror -Og")
 
 include(FetchContent)
 
-find_package(CURL 7.81.0 EXACT REQUIRED)
+find_package(CURL REQUIRED)
```

Comment:
It is not the part of this PR. Please revert it and re-create a PR for curl version issue.